tests/test_command_parse.py

```diff
@@ -29,7 +29,15 @@ def setUp(self):
 import scrapy
 from scrapy.linkextractors import LinkExtractor
 from scrapy.spiders import CrawlSpider, Rule
+from scrapy.utils.test import get_from_asyncio_queue
 
+class AsyncDefAsyncioSpider(scrapy.Spider):
+
+    name = 'asyncdef{self.spider_name}'
```

Comment:
This is not an f-string :)
But I don't think you need a dynamic value here anyway.